migrating.md

```diff
@@ -110,6 +110,7 @@ Common issues on RequireJS pages:
 - JS error like `$(...).something is not a function`: this indicates there's a missing dependency. Typically "something" is either `select2` or a jQuery UI widget like `datepicker`. To fix, add the missing dependency to the module that's erroring.
 - Missing functionality, but no error: this usually indicates a missing knockout binding. To fix, add the file containing the binding to the module that applies that binding, which usually means adding `hqwebapp/js/knockout_bindings.ko` to the page's main module.
 - JS error like `something is not defined` where `something` is one of the parameters in the module's main function: this can indicate a circular dependency. This is rare in HQ. Track down the circular dependency and see if it makes sense to eliminate it by reorganizing code. If it doesn't, you can use [hqRequire](https://github.com/dimagi/commcare-hq/commit/15b436f77875f57d1e3d8d6db9b990720fa5dd6f#diff-73c73327e873d0e5f5f4e17c3251a1ceR100) to require the necessary module at the point where it's used rather than at the top of the module using it.
+- JS error like `x is not defined` where `x` is a third-party module, which is the dependency of another third party module `y` and both of them are non RequireJs modules. You may get this intermittent error when you want to use `y` in the migrated module and `x` and `y` does not support [AMD](https://requirejs.org/docs/whyamd.html). You can fix this using [shim](https://www.devbridge.com/articles/understanding-amd-requirejs#To-shim-or-not-to-shim) or [hqRequire](https://github.com/dimagi/commcare-hq/commit/15b436f77875f57d1e3d8d6db9b990720fa5dd6f#diff-73c73327e873d0e5f5f4e17c3251a1ceR100). Example of this could be `d3` and `nvd3`
```

Comment:
Can you make "Example" link to the example code?

Reply:
Yes Sure,